build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     ext {
         opensearch_group = "org.opensearch"
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
-        opensearch_version = System.getProperty("opensearch.version", "2.2.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
```

Comment:
Sorry, realize the branch is 2.x after approving. Can you change it to `3.0.0-SNAPSHOT` ?

Reply:
Already changed it to 3.0.0. Does it need to be 2.3.0-SNAPSHOT?